app/jobs/bcc_post.rb

```diff
@@ -21,6 +21,8 @@ def execute(args)
 
   def send_to(targets, targets_key, params, sender)
     targets.each do |target|
+      params["raw"] = params["raw"].gsub(/{username}/i, target)
+      params["raw"] = params["raw"].gsub(/{@username}/i, "@" + target)
```

Comment:
I see you added this case but there's no test for this

Reply:
Added a second test case for @ mentions.